internal/generate/clusterserviceversion/bases/clusterserviceversion.go

```diff
@@ -70,6 +70,9 @@ func (b ClusterServiceVersion) GetBase() (base *v1alpha1.ClusterServiceVersion,
 		base = b.makeNewBase()
 	}
 
+	// Add sdk stamps to csv
+	addSDKstamps(base)
+
```

Comment:
Adding SDK stamps while creating base CSV as these are constant values which user should not change and should be present always.